lld/test/ELF/arm-thumb-jump8-11.s

```diff
@@ -0,0 +1,30 @@
+# RUN: llvm-mc -triple thumbv6m-arm-eabi --filetype=obj %s -o %t.obj
```

Comment:
You'll need a `# REQUIRES: arm` at the top of the file to stop the test failing on targets that haven't enabled the ARM target.

Apologies for not spotting yesterday.

Reply:
Thanks, Fixed